src/main/resources/templates/templateDslBuilder.ftlh

```diff
@@ -1,7 +1,68 @@
 <#assign constructor = false>
 <#macro writeAttrField field>
   <#if field.functionByType??>
-    <#if field.fieldType == "boolean">
+    <#if field.fieldType == "String[]"
```

Comment:
Maybe is better to create a fieldType Array and then with a subFieldType that is the type of array?

And if fieldType == Array then send it to a new method to build to keep "low logic" in the method.